src/instructlab/config/init.py

```diff
@@ -320,14 +327,17 @@ def match_profile_based_on_gpu_count(
 
 def check_if_configs_exist(fresh_install) -> bool:
     if exists(DEFAULTS.CONFIG_FILE):
-        overwrite = click.confirm(
-            f"Found {DEFAULTS.CONFIG_FILE}, do you still want to continue?"
-        )
+        click.echo(f"Existing config file was found in:\n    {DEFAULTS.CONFIG_FILE}")
+        overwrite = click.confirm("Do you still want to continue?")
```

Comment:
Same comment here that I am not sure whether the newline could cause some people to not make the connection.